pytket/phir/phirgen.py

```diff
@@ -47,12 +42,12 @@ def genphir(inp: list[tuple[list[int], list[Shard], float]]) -> str:
     Args:
         inp: list of shards
     """
-    phir = {
+    phir: dict[str, Any] = {
```

Comment:
No better types we can give than `Any`?

Reply:
With the current approach, i.e., constructing Python dict's instead of a `PHIRModel` object, the best we can do is `Any`. Someday, we can consider directly constructing `PHIRModel` but I found it difficult to go that route in my first attempt.